node/pkg/fetcher/accumulator.go

```diff
@@ -21,16 +23,33 @@ func (a *Accumulator) Run(ctx context.Context) {
 	a.cancel = cancel
 	a.isRunning = true
 
+	// dummy channel to signal accumulator workers to batch insert data to db
+	jobChannel := make(chan struct{}, 100)
```

Comment:
why do the jobChannel need 100 buffers? and how did you come up with number 100?

Reply:
It's actually random. I don't expect the buffer size to go over 3 so I could use `ACCUMULATOR_WORKER_COUNT` as the buffer size but it was just a safety measure. I'll change it to `ACCUMULATOR_WORKER_COUNT*2`, what do you think?

Reply:
I think we can remove buffer here, because it won't be stacked up in the channel but rather consumed immediately